seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/rest/servlet/AllNodeLogServlet.java

```diff
@@ -66,7 +66,7 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             FormatType formatType = FormatType.fromString(req.getParameter("format"));
             switch (formatType) {
                 case JSON:
-                    writeJson(resp, logService.allNodeLogFormatHtml(jobId));
+                    writeJson(resp, logService.allNodeLogFormatJson(jobId));
```

Comment:
cc @zhangshenghang

Reply:
We missed add some test case in https://github.com/apache/seatunnel/blob/06df51bcbbb3294f40ebb5607c70bc031c5175f8/seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/joblog/JobLogIT.java#L56. Could you help to add test case @showfine ?

Reply:
Sure.

Reply:
> cc @zhangshenghang and @liugddx

Yes, he's right. thks